lighthouse-cli/test/fixtures/perf/animations.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```

Comment:
license :)

Reply:
Will do.

A lot of other smoke tests don't have a license.  Is there a reason for that or am I not the first person to forget?

Reply:
you're definitely not the first :) a few forgot and then those were probably copy/pasted where the author definitely won't notice.